app/helpers/application_helper.rb

```diff
@@ -21,6 +21,11 @@ def logo_image
     mentor_or_student_logo
   end
 
+  def scoped_image_tag(source, options = {})
+    scoped_source = "#{Rails.application.config.settings.application_name}/#{source}"
+    image_tag(scoped_source, options)
+  end
+
```

Comment:
Ik vind het mooi, ik zit me alleen nog af te vragen, willen we alles (dus ook dingen zoals plaatjes) binnen die organization directory houden? Ik denk eigenlijk van wel. Het mooie zou zijn als je gewoon een bestand zoals `config/projects/u-can-act/assets/images/organization_logo.png` kan hebben. En dat je die dan include met `scoped_image_tag('organization_logo')`.

Nu doen we namelijk dingen op twee plaatsen: we hebben die scoped image tag die zegt van in welke directory je de plaatjes kan vinden, en die config.settings.logo.fallback_logo settings die zeggen hoe het plaatje heet. Maar slechts één van de twee is nodig: de enige reden dat we b.v. de naam van het fallback logo in een organization setting hadden staan is omdat alle plaatjes in dezelfde directory stonden. Als iedere organization haar eigen directory met plaatjes heeft, dan kun je het organization logo gewoon `organization_logo.png` noemen, en dan is het ook transparanter omdat de bestanden in de verschillende projectdirectories dan altijd gewoon dezelfde naam en functie hebben.

Wat ik denk dat gewoon te doen is is dat je ergens in die settings initializer, dat je daar die config/projects/<project_name>/assets/images directory toevoegt aan de image asset pipeline (of de assets directory aan de assets pipeline). Ik weet niet precies hoe je dat doet, maar ik gok dat dat wel kan. En dat zou vet hip zijn want dan zijn alle organisatiespecifieke dingen dus echt ondergebracht in 1 directory, en dan hoef je maar 1 keer cmd+c cmd+v te doen om een nieuwe organisatie aan te maken.